content/rooms/v11.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Room Version 11
+type: docs
+weight: 100
+---
+
+This room version builds on [version 10](/rooms/v10) while clarifying redaction
+rules.
+
+## Client considerations
+
+Clients should no longer depend on the `creator` property of `m.room.create` events.
+
+Clients should note that the format of `m.room.redaction` events has been modified
+and look for the `redacts` key under `content` instead of a top-level event property.
+
+Clients should note that the `third_party_invite` key of `m.room.member` events
+is no longer redacted, *but* will only contain the `signed` key after redaction.
+
+Clients which implement the redaction algorithm locally should refer to the
+[redactions](#redactions) section below for a full overview.
+
+## Server implementation components
+
+{{% boxes/warning %}}
+The information contained in this section is strictly for server
+implementors. Applications which use the Client-Server API are generally
+unaffected by the intricacies contained here. The section above
+regarding client considerations is the resource that Client-Server API
+use cases should reference.
+{{% /boxes/warning %}}
+
+This room version updates the redaction algorithm and modifies how servers should
+create `m.room.create` and `m.room.redaction` events.
+
+Room version 11 is based upon room version 10 with the following considerations.
+
+### Event format
+
+The event format is unchanged by this room version. See [below](#event-format-1)
```

Comment:
It is incongruous that we start the section saying that events have not changed, and then go on to describe how they change.

Maybe: "The *core* event format is unchanged ... However, this room version changes the *content* of some event types".

Reply:
Also: https://pr1604--matrix-spec-previews.netlify.app/rooms/v11/#event-format-1 includes "redacts" at the top level, which *is* changing.

Reply:
Right. I'm trying to get rid of that now, but it is proving difficult!

Reply:
I think I cleaned this up. I had to copy & paste a bit with the event formats since you can't seem to remove fields from a JSONschema?